files/image_config/copp/copp_cfg.j2

```diff
@@ -11,7 +11,12 @@
 	    "queue4_group1": {
 		    "trap_action":"trap",
 		    "trap_priority":"4",
-		    "queue": "4"
+		    "queue": "4",
+		    "meter_type":"packets",
+		    "mode":"sr_tcm",
+		    "cir":"6000",
```

Comment:
Any reason to rate limit BGP while the problem you mentioned was with DHCP flooding? How is this value of 6000 derived?

Reply:
@dgsudharsan , 6000 is a legacy value we set for IP2me packets which has satisfied our trap requirements.